### PR TITLE
Feature/waqi stations dynamic

### PR DESCRIPTION
**Pull request by hucksy**

### CODE REVIEW

**Comment by hucksy** [comment]
yooo bunch of changes for the station stuff. I also messed around with adding another logger that's separate from the airflow one, though I think it needs some more tweaking.

File: services/etl/dags/api_interface/test_coroutine.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def coroutine(func):
```
Comment (myshkins) [inline services/etl/dags/api_interface/test_coroutine.py]: what's goin on with this file?

File: services/etl/dags/api_interface/get_readings_waqi.py [hunk 2]
```diff
@@ -1,37 +1,93 @@
...
+    """query database for list of all WAQI station and returns list of all station ids"""
+    sql_stmt = select(Waqi_Stations.station_id)
+    with get_db() as db:
+        stations_result = db.execute(sql_stmt).all()
+    station_list = [station.station_id for station in stations_result]
+    return station_list
+
+
+def init_urls():
+    """loops through all waqi stations and asynchronously requests data for each."""
```
Comment (myshkins) [inline services/etl/dags/api_interface/get_readings_waqi.py]: is the doc string mixed up here?

File: services/etl/requirements.txt [hunk 3]
```diff
@@ -1,118 +1,11 @@
+aiohttp==3.8.3
+apache-airflow==2.5.1
```
Comment (myshkins) [inline services/etl/requirements.txt:2]: is the airflow import required? it should already be there with the docker image right?

**Review by myshkins — APPROVED** [review]
looks awesome dude! the logger and async stuff is really cool

File: services/etl/dags/api_interface/get_readings_waqi.py (hunk 2 above)
Comment (hucksy) [reply services/etl/dags/api_interface/get_readings_waqi.py]: roiight, will update

File: services/etl/requirements.txt (hunk 3 above)
Comment (hucksy) [reply services/etl/requirements.txt:2]: right, will remove

**Pull request merged**